exercises/linked-list/README.md

```diff
@@ -82,3 +82,23 @@ The setup files are in the [exercise-3](https://github.com/FACN7/interview-prep/
 Have you tried the <a href="https://www.geeksforgeeks.org/write-a-c-function-to-print-the-middle-of-the-linked-list/">fast & slow</a> method?
```

Comment:
yes,
also solved it using a stack

Reply:
oh sorry,
used a stack to solve a different exercise(reverse a linkedList exercise)